docs/getting-started/using-gnokey.md

```diff
@@ -0,0 +1,777 @@
+
+```bash
+gnokey maketx call \
+-pkgpath "gno.land/r/demo/wugnot" \
+-func "Deposit" \
+-send "1000ugnot" \
+-gas-fee 10000000ugnot \
+-gas-wanted 2000000 \
+-broadcast \
+-chainid portal-loop \
+-remote "https://rpc.gno.land:443"" \
+mykey
+```
+
+In this command, we have specified three main things:
+- The path where the realm lives on-chain with the `-pkgpath` flag
+- The function that we want to call on the realm with the `-func` flag
+- The amount of `ugnot` we want to send to be wrapped, using the `-send` flag
+
+Apart from this, we have also specified the Portal Loop chain ID, `portal-loop`,
+as well as the Portal Loop remote address, `https://rpc.gno.land:443`.
+
+After broadcasting the transaction, we can verify that we have the amount of `wugnot` we expect. We
+can call the `BalanceOf()` function in the same realm:
+
+```bash
+gnokey maketx call \
+-pkgpath "gno.land/r/demo/wugnot" \
+-func "BalanceOf" \
+-args "<your_address>" \
+-gas-fee 10000000ugnot \
+-gas-wanted 2000000 \
+-broadcast \
+-chainid portal-loop \
+-remote "https://rpc.gno.land:443" \
+mykey
+```
+
+If everything was successful, we should get the following output:
+
+```
+(1000 uint64)
+
+OK!
+GAS WANTED: 2000000
+GAS USED:   396457
+HEIGHT:     64839
+EVENTS:     []
+```
+
+At the top, you will see the output of the transaction, specifying the value and
+type of the return argument.
+
+In this case, we used `maketx call` to call a read-only function, which simply
+checks the `wugnot` balance of a specific address. This is discouraged, as
+`maketx call` actually uses gas. To call a read-only function without spending gas,
+check out the `vm/qeval` query in the [ABCI queries section](#vmqeval).
+
+### `Send`
+
+We can use the `Send` message type to access the TM2 [Banker](../concepts/stdlibs/banker.md)
+directly and transfer coins from one Gno address to another.
+
+Coins, such as GNOTs, are always formatted in the following way:
+
+```
+<amount><denom>
+100ugnot
+```
+
+For this example, let's transfer some GNOTs. Just like before, we can configure
+our `maketx send` subcommand:
+```bash
+gnokey maketx send \
+-to g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5 \
+-send 100ugnot \
+-gas-fee 10000000ugnot \
+-gas-wanted 2000000 \
+-broadcast \
+-chainid portal-loop \
+-remote "https://rpc.gno.land:443" \
+mykey
+```
+
+Here, we have set the `-to` & `-send` flags to match the recipient, in this case
+the publicly-known `test1` address, and `100ugnot` for the coins we want to send,
+respectively.
+
+To check the balance of a specific address, check out the `bank/balances` query
+in the [ABCI queries section](#bankbalances).
+
+### `Run`
+
+With the `Run` message, you can write a snippet of Gno code and run it against
+code on the chain. For this example, we will use the [Userbook realm](https://gno.land/r/demo/userbook),
+which simply allows you to register the fact that you have interacted with it.
+It contains a simple `SignUp()` function, which we will call with `Run`.
```

Comment:
Please see my earlier comment about building out the Realm for this tutorial

Reply:
https://github.com/gnolang/gno/pull/2226#discussion_r1733341235